lib/db/init.ts

```diff
@@ -0,0 +1,38 @@
+import {db} from "../../lib/db";
+
+export function initDB() {
+    const inELW: number = 309;
+    const inECS: number = 529;
+
+    // clear the database
+
+    const clearBuildings = db.prepare("DELETE FROM buildings");
+    const clearLockers = db.prepare("DELETE FROM lockers");
+    const clearUsers = db.prepare("DELETE FROM users");
+    const clearRegistrations = db.prepare("DELETE FROM registrations");
```

Comment:
What are these for?

Reply:
I was thinking this would be used at the beginning of a term so the lockers would be unregistered anyway

Reply:
Best to leave unused code out until you need it.